ppr-ui/src/components/common/CertifyInformation.vue

```diff
@@ -28,8 +28,8 @@
             <!-- Table Body -->
             <tbody v-if="registeringParty.length > 0">
               <tr v-for="(item, index) in registeringParty" :key="`${item}: ${index}`" class="party-row">
-                <td class="list-item__title title-text" style="padding-left:30px">
-                  <v-icon class="v-icon mt-n2">mdi-account</v-icon><span>{{ legalName }}</span>
+                <td class="list-item__title title-text icon-text" style="padding-left:30px">
```

Comment:
Simple generic class that aligns the icons with the text associated to it.

Reply:
We should cleanup inline styles at some point.

Reply:
100%